Add Support for Embedding TriggerBindingSpec in EL (#548)
tektoncd/triggers: docs/eventlisteners.md

- `name` - (Optional) a valid
  [Kubernetes name](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set)
- [`interceptors`](#interceptors) - (Optional) list of interceptors to use

> - `bindings` - A list of names of `TriggerBindings` to use, TriggerBindingsSpec can also be embedded.

Reviewer: ```suggestion
- `bindings` - A list of references to `TriggerBindings` or embedded TriggerBindingsSpecs to use.
```

Reviewer: Can we also provide a quick example snippet here showing a `ref` and an embedded spec?

Author: I have given here: https://github.com/tektoncd/triggers/pull/548/commits/267b026ee58b3149d7cceba5bdb98a7b73a426ba#diff-be0d7f59d8d71d22c7dcc42f4be3eb3fR165
